core/src/main/java/nu/yona/server/subscriptions/service/UserService.java

```diff
@@ -169,8 +170,8 @@ private void addMandatoryGoals(User userEntity)
 
 	private void addGoal(User userEntity, ActivityCategoryDTO category)
```

Comment:
I guess this method should have been renamed already longer ago to `addMandatoryNoGoGoal`...